tests/pipelines/test_video2video.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import unittest
+
+from modelscope.models import Model
+from modelscope.outputs import OutputKeys
+from modelscope.pipelines import pipeline
+from modelscope.utils.constant import DownloadMode, Tasks
+from modelscope.utils.test_utils import test_level
+
+
+class Video2VideoTest(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.task = Tasks.video_to_video
+        self.model_id = 'damo/Video-to-Video'
```

Comment:
`'damo/Video-to-Video'` seems to be unreachable, please make it publicly visible